src/app/core/feedback/feedback.controller.spec.js

```diff
@@ -21,20 +21,9 @@ describe('Feedback Controller', () => {
 
 	const clearDB = () => Feedback.deleteMany({});
 
-	const spec = [];
-
 	before(clearDB);
 	after(clearDB);
 
-	before(() => {
-		// Reduce to insert them in order
-		return spec.reduce((promise, spec) => {
-			return promise.then((res) => {
-				return new Feedback(spec).save().then(Array.prototype.concat.bind(res));
-			});
-		}, Promise.resolve([]));
-	});
-
```

Comment:
SonarQube flagged this because at this point, the `spec` list will always be empty. This `spec` isn't used anywhere else in the file, as it's re-declared in each of the tests.